0107-preventing-gaming-within-the-mobile-network.md

```diff
@@ -0,0 +1,81 @@
+
+### Maximum Asserted Distance Difference
+For indoor and outdoor Wi-Fi access points, Hotspot locations are verified using external services like Skyhook. If the asserted distance is greater than 1000 meters away from the external service location, the trust score will be set to `0.00`.
+
+Further, in order for access points to be eligible for boosted Service Provider rewards defined in HIP-84, the asserted distances must be 30 meters or less than the reported location from external services. In instances where the difference is greater than 30 meters, the access point will still be eligible for PoC rewards, but not boosted rewards.
+
+### Picture Requirements of Outdoor Wi-Fi
+This proposal also requires deployers asserting a new location for outdoor Wi-Fi access points to also submit a picture of the Access Point installation. This picture will be submitted directly to the Hotspot Maker and must show the height of the installation from the ground level.
+
+## Rejection Due to Gaming Concerns
+The following is a list of gaming reasons that the CPI/Service Provider may deny a submission or deactivate a device:
+
+- Radio/Wi-Fi AP is not in the same location as the submission data states. This includes:
+    - Height mismatch
+    - Location mismatch
+    - Spoofing of location
+
+- False, inaccurate, or deceptive images/data submitted
+
+- Fraud
+    - Including subscriber abuse of subscribers signing up, offloading data, and then canceling the subscriber plan prior to grace period end date
+    - Consistently offloading data of subscribers who signed up with fraudulent information, such as payment source
+
+
+## Abuse of Power
+If it is identified that Service Providers are abusing their power to turn off MOBILE equipment without valid gaming reasons, or do not adhere to the requirements of this HIP, a governance proposal may be raised by the MOBILE Working Group to slash an amount of the Service Providers stake. The proposal must be voted on by the MOBILE network with veMOBILE and pass by at least 67%.
+
+## Drawbacks
+
+Deployers who have an outdoor Wi-Fi Access Point asserted higher than 100 feet will need to update the assertion height to a maximum of 100 feet.
+
+## Rationale and Alternatives
+An alternative would be to do nothing and allow gaming to continue.
+
+## Implementation Phases
+
+For HIP implementation, Helium Mobile and Nova Labs has agreed to do the following:
+
+- Update the Helium Builder app to require a picture be uploaded when new assertions are done for outdoor Wi-Fi access points. This feature also needs to be backported to the Helium Wallet app since assertions can occur in this application.
```

Comment:
The helium wallet app needs to be updated to require this as well. And the elevation.

Reply:
I think it says that but open to edits.